spec/controllers/equipment_models_controller_spec.rb

```diff
@@ -19,42 +19,34 @@
         it_behaves_like 'successful request', :index
       end
 
-      context 'with @category set' do
-        it 'populates an array of of active category-type equipment models' do
+      it 'defaults to all active equipment models' do
```

Comment:
See, here is where I struggle with our current assertion. How does this, in any way, validate that we're querying for Equipment Models of all Categories. Sure, we can rely on the assertion that the `active` scope is queried for the active part, but it still feels like there's something missing to me.